site/docs/master/disaster-case.md

```diff
@@ -14,10 +14,26 @@ If you periodically back up your cluster's resources, you are able to return to
 
 1.  A disaster happens and you need to recreate your resources.
 
-1.  Update the Velero server deployment, adding the argument for the `server` command flag `restore-only` set to `true`. This prevents Backup objects from being created or deleted during your Restore process.
+1.  Update your backup storage location to read-only mode (this prevents backup objects from being created or deleted in the backup storage location during the restore process):
+
+    ```bash
```

Comment:
Would it be useful to include our own command to do these operations later? Something like `velero backup-locations edit mylocation --read-only/--read-write`? I'd think we wouldn't need a controller for it, and it's a little less for users to type.

Reply:
Yeah, agree it would be nice to have.  We don't really have any `edit` commands right now so I'd want to think a little about what the UX should be.